lib/Differentiator/ReverseModeVisitor.cpp

```diff
@@ -1938,12 +1923,28 @@ Expr* ReverseModeVisitor::getStdInitListSizeExpr(const Expr* E) {
           if (!baseDerivative->getType()->isPointerType())
             baseDerivative =
                 BuildOp(UnaryOperatorKind::UO_AddrOf, baseDerivative);
-          CallArgDx.insert(CallArgDx.begin(), baseDerivative);
-          revForwAdjointArgs.insert(revForwAdjointArgs.begin(), baseDerivative);
+          CallArgDx.push_back(baseDerivative);
+          revForwAdjointArgs.push_back(baseDerivative);
         }
       }
     }
 
+    for (std::size_t i = static_cast<std::size_t>(isMethodOperatorCall),
+                     e = CE->getNumArgs();
```

Comment:
```suggestion
    for (std::size_t skip_this = static_cast<std::size_t>(isMethodOperatorCall), i = skip_this,
                     e = CE->getNumArgs();
```
and below you can use that variable in `PVD`...

Reply:
Done. Is that what you had in mind?

Reply:
Yeah.